spec.html

```diff
@@ -3720,6 +3731,8 @@
           </tbody>
         </table>
       </emu-table>
+      <!-- If ToObject is ever made hookable, we need to alert the editors of the HTML Standard as
+           that'll effect the security model of the Location exotic object. -->
```

Comment:
Who is the intended audience for such a note?  ECMAScript language designers or implementors?